features/main/url_encoded_id.feature

```diff
@@ -3,24 +3,41 @@ Feature: Allowing resource identifiers with characters that should be URL encode
   As a client software developer
   I need to be able to set and retrieve these resources with the URL encoded ID
 
+  # Symfony\Component\Routing\Generator\UrlGenerator::generate uses rawurlencode so @id/iri will be encoded.
   @createSchema
   Scenario Outline: Get a resource whether or not the id is URL encoded
     Given there is a UrlEncodedId resource
     And I add "Content-Type" header equal to "application/ld+json"
-    When I send a "GET" request to "<url>"
+    When I send a "GET" request to "/url_encoded_ids/<id>"
     Then the response status code should be 200
     And the JSON should be equal to:
     """
     {
         "@context": "/contexts/UrlEncodedId",
-        "@id": "/url_encoded_ids/%25encode:id",
+        "@id": "/url_encoded_ids/%25./encode:001",
         "@type": "UrlEncodedId",
-        "id": "%encode:id"
+        "id": "%./encode:001"
     }
     """
     Examples:
-      | url                              |
-      | /url_encoded_ids/%encode:id      |
-      | /url_encoded_ids/%25encode%3Aid  |
-      | /url_encoded_ids/%25encode:id    |
-      | /url_encoded_ids/%encode%3Aid    |
+      | id                     |
+      | %25%2E%2Fencode%3A001  |
```

Comment:
@alanpoulain  this is the only identifier that feels right to me as this is the Resource identifier. It should not have non-urlencoded parts.